server/user/user.js

```diff
@@ -57,16 +69,37 @@ class User {
     }
   }
 
-  static async getUserAccount ({ email, tokenUser }) {
+  /**
+   * Get details of a user's account
+   * @param {Object} params
+   * @param {String} params.email Email address for user
+   */
+  static async getUserAccount ({ email }) {
     try {
-      if (tokenUser.user !== email) {
-        throw new Error('Cannot retrieve info for another account');
-      }
-      const user = await UserModel.findOne({ email });
-      if (user === null) {
+      const userAccount = await UserModel.findOne({ email });
+      if (userAccount === null) {
         throw new Error(`User with email ${email} was not found`);
```

Comment:
Do we want to return nothing in this case? or should we reject, or is the best practice to allow the calling method to fall into `.catch()`?

Reply:
The practice I prefer is to only return a successful response if it's expected that there may be no data to return. In this case, you'd only call getUserAccount if you are expecting one, so a 4xx error would be appropriate, so the front-end can know to try again with different credentials.

However, I think right now the error middleware is defaulting to 5xx errors, so I need to go back through and update the errors so far in these files with propers codes.